chia/full_node/block_store.py

```diff
@@ -52,6 +52,8 @@ async def create(cls, db_wrapper: DBWrapper):
             # peak. The "key" field is there to make update statements simple
             await self.db.execute("CREATE TABLE IF NOT EXISTS current_peak(key int PRIMARY KEY, hash blob)")
 
+            # If any of these indices are altered, they should also be altered
+            # in the chia/cmds/db_upgrade.py file
```

Comment:
```suggestion
            # in the chia/cmds/db_upgrade.py file, and the database version should be incremented
```

Reply:
technically, it's sometimes possible to slightly alter indices without breaking compatibility with the existing inserts and lookups. That's mainly what I had in mind when writing this comment.